Inc/GamePad.h

```diff
@@ -228,7 +228,32 @@ namespace DirectX
             ButtonState leftTrigger;
             ButtonState rightTrigger;
 
-            ButtonStateTracker() throw() { Reset(); }
+            ButtonStateTracker() noexcept :
```

Comment:
This seems less good than the previous implementation, as it duplicates knowledge of all the individual button fields in an additional location.

Reply:
I know. ``Reset`` doesn't make VS 2017 (15.7) ``/analyze`` happy though :(